guides/plugins/plugins/administration/add-new-tab.md

```diff
@@ -223,13 +223,13 @@ As shown in the previous example, your custom component is expected to be in a d
 
 ```text
 administration
-├── page
-│   └── sw-product-detail
+├── src
+│   └──page
 │       └── sw-product-detail
 │           ├── index.js
 │           └── sw-product-detail.html.twig
-├── view
-│   └── sw-product-detail-custom
+│       └── view
```

Comment:
It [seems](https://github.com/shopware/docs/blob/d943dd224897e59fe4fdc05b28d44fa3a3b00924/guides/plugins/plugins/administration/add-new-tab.md#creating-your-new-component) like `view` needs to be on the same level as `page` - please, revert changes in indentation for `view`, and keep the indentation as it is for `sw-product-detail-custom` (as it was fixed correctly).